cluster.go

```diff
@@ -115,7 +150,8 @@ func listCloud() (*Cloud, error) {
 func createCluster(name string, nodes int, lifetime time.Duration) error {
 	vmNames := make([]string, nodes, nodes)
 	for i := 0; i < nodes; i++ {
-		vmNames[i] = fmt.Sprintf("%s-%0.4d", name, i)
+		// Start instance indexing at 1.
```

Comment:
Alternately, we could change `roachperf` and other tools as well.

Reply:
The original reason to move from 0-indexing to 1-indexing was to have hostnames match node IDs (when nodes are brought up in sequential order).
I think this is still desirable. If the nodes are brought up in machine-name order (difficult with concurrency, but we could tweak things to make it happen again), it's still less misleading to have it match.